docs/medical-api/more-info/webhooks.mdx

```diff
@@ -359,3 +359,69 @@ Example payload:
   ]
 }
 ```
+
+### Bulk Document Download URLs
+
+If you want to download all of a patient's documents, especially when they have a large volume of documents, you can start a [Bulk Get Document URL query](/medical-api/api-reference/document/bulk-get-document-url). We’ll send a webhook message with all the downloadable URLs for all the patient's documents.
+
```

Comment:
nit: we could be explicit here and add a separate sentence instructing the dev to use the `url` property to download the doc - perhaps:

You can use the value of the `url` property in the returned `documents` objects to download the files.